client/my-sites/purchase-product/controller.js

```diff
@@ -27,7 +27,9 @@ import {
 	PRODUCT_JETPACK_SEARCH_MONTHLY,
 	PRODUCT_WPCOM_SEARCH,
 	PRODUCT_WPCOM_SEARCH_MONTHLY,
+	isJetpackProductSlug,
```

Comment:
`isJetpackProductSlug` relies on `JETPACK_PRODUCTS_LIST` which uses `JETPACK_SEARCH_PRODUCTS`. The latter is defined as:
```
export const JETPACK_SEARCH_PRODUCTS = <const>[
	PRODUCT_JETPACK_SEARCH,
	PRODUCT_JETPACK_SEARCH_MONTHLY,
	PRODUCT_WPCOM_SEARCH,
	PRODUCT_WPCOM_SEARCH_MONTHLY,
];
```

🙃